test/common.dart

```diff
@@ -5,12 +5,15 @@
 import 'package:code_builder/code_builder.dart';
 import 'package:dart_style/dart_style.dart';
 
-final DartFormatter _dartfmt = DartFormatter();
```

Comment:
It's hard to discern the use cases of this, but it may make sense to keep this as a top-level so we're not reinstantiating every time we call `_format`.

Reply:
I admit this is maybe not the best API design, but `DartFormatter` has always been a very lightweight, ephemeral class. It's basically so that you don't have to pass a big pile of arguments to `format()`. It's fine to construct a new one on every invocation. And, arguably, it's better to do so because it ensures that the tiny amount of mutable state inside the class isn't shared across unrelated formats (https://github.com/dart-lang/dart_style/issues/1337).

Reply:
Ack, this is fine then. Thanks!